spring-cloud-kubernetes-client-autoconfig/src/test/java/org/springframework/cloud/kubernetes/client/example/AppWithoutApiClient.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Sorry for the confusion, I would just put this as an inner class in your test since its the only test that needs it at the moment.